helios-system-tests/src/main/java/com/spotify/helios/system/ResourcesTest.java

```diff
@@ -99,13 +99,13 @@ public void testClient() throws Exception {
 
     try (final DockerClient docker = getNewDockerClient()) {
 
-      final ContainerConfig containerConfig =
-          docker.inspectContainer(taskStatus.getContainerId()).config();
+      final HostConfig hostConfig =
+          docker.inspectContainer(taskStatus.getContainerId()).hostConfig();
 
-      assertEquals(MEMORY, containerConfig.memory());
-      assertEquals(MEMORY_SWAP, containerConfig.memorySwap());
-      assertEquals(CPU_SHARES, containerConfig.cpuShares());
-      assertEquals(CPUSET, containerConfig.cpuset());
+//      assertEquals(MEMORY, hostConfig.memory());
```

Comment:
For some reason, the memory limits aren't being on `docker.createContainer()` [here][1].

  [1]: https://github.com/spotify/helios/pull/590/files#diff-12ef098bf3b4de9a99db3d575871cb50R227